modules/custom/engineering_profile_helper/modules/engineering_magazine/templates/magazine-curtain-block.html.twig

```diff
@@ -1,10 +1,23 @@
 <div id="magazine-curtain" class="magazine-curtain hero-curtain hero-scroll">
 
+	<div class="curtain-image">
+		<img src="{{ curtain_content.media_url }}"/>
+	</div>
+
 	<div class="curtain-content">
-		<h1>This is my curtain.</h1>
 
-		<p>Curtain content:
-			{{ curtain_content }}</p>
+		<a href="{# curtain_content.issue_url #}" class="engineering-accent-color__background">
```

Comment:
Is it okay this href has commented out variable in it?

Reply:
good catch.

Reply:
fixed.